### PR TITLE
Feedback on v.0.1.1

### PR DESCRIPTION
**Pull request by avandeursen**
- Updated abstract
- Various small textual improvements throughout the text
- Cleaned up bibtex entries

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -414,7 +414,10 @@
 \begin{abstract}
 %\boldmath
 
-Continuous Deployment is a hot topic. It holds much promise for developing and deploying software applications. However both software and release engineers face various challenges when adopting this practice. In our view, the biggest technical challenge in the development and deployment of software (which uses SQL databases to store persistent data), is evolving the database schema of SQL databases without disrupting the applications connected to the database. In this paper we explore tools, and approaches to address this problem.
+Continuous Deployment is an important enabler rapid delivery of business value and early end user feedback. While frequent code deployment is well understood, the impact of frequent change on persistent data is less understood and supported. SQL schema evolutions in particular can make it expensive to deploy a new version, and may even lead to downtime if schema changes can only be applied by blocking operations.
```
Comment (michaeldejong) [inline paper.tex]: Do we need to add an `of` in here?

> Continuous Deployment is an important enabler **of** rapid delivery of business value...

File: paper.tex [hunk 2]
```diff
@@ -414,7 +414,10 @@
...
+
+In this paper we study the problem of continuous deployment in the presence of database schema evolution in more detail. We identify a number of shortcomings to existing solutions and tools, mostly related to avoidable down time and support for foreign keys. We propose a novel approach to address these problems, and provide an open source implementation. Initial evaluation suggests the approach is effective and sufficiently efficient.
```
Comment (michaeldejong) [inline paper.tex]: Rename "down time" to "downtime".

File: paper.tex [hunk 3]
```diff
@@ -447,55 +450,61 @@ \section{Introduction} %(0.7)
...
+
+
+When service windows are short or even non-existent, these two problems make releasing new versions of applications frequently problematic for release engineers. To address this problem we aim to find or implement a tool which  allows its users to not only alter their database schema in a non-blocking fashion, but also do this without affecting older and still running applications.
```
Comment (michaeldejong) [inline paper.tex]: Double space.

File: paper.tex [hunk 4]
```diff
@@ -447,55 +450,61 @@ \section{Introduction} %(0.7)
...
+These are both tools which closely follow Approach~1 descriged earlier (Section~\ref{sec:cd}). They create a copy of a table (including its contents) and record modifications to records in the original table using triggers into a special ``deltas'' table. Once they have copied the original table and its contents, they apply the specified schema operations to the ghost table before proceeding to replay the recorded modifications from the ``deltas'' table onto the ghost table. When the ``deltas'' table is almost empty, these tools acquire an exclusive write lock on the original table --- like \textit{TableMigrator} does --- and apply the last of the recorded modifications before atomically switch the names of both tables.
```
Comment (michaeldejong) [inline paper.tex]: Typo: described

File: paper.tex [hunk 5]
```diff
@@ -447,55 +450,61 @@ \section{Introduction} %(0.7)
...
 
 \subsection{Limitations} 
 
 \begin{itemize}
-  \item{None of the tools used in practice support foreign keys and primarily focus on MySQL databases.}
-  \item{These tools are very limited in the sense that they often only support making small changes to the existing database schema. This forces software and release engineers to often release intermediate versions of the application, as the schema is altered in several steps.}
+  \item None of the tools used in practice support foreign keys
```
Comment (michaeldejong) [inline paper.tex]: Missing trailing dot.

File: paper.tex [hunk 6]
```diff
@@ -447,55 +450,61 @@ \section{Introduction} %(0.7)
...
+  \item The tools primarily focus on MySQL databases.
+  \item These tools are limited in the sense that they typically only support making small changes to the existing database schema. This forces software and release engineers to often release intermediate versions of the application, as the schema is altered in several steps.
 \end{itemize}
 
-\section{Our approach} %(1.0)
+\section{Proposed Approach} %(1.0)
```
Comment (michaeldejong) [inline paper.tex:507]: Yes, that sounds better!

**Comment by michaeldejong** [comment]
I'll update this PR with my minor remarks, and then merge it...

**Comment by avandeursen** [comment]
Excellent!

**Pull request merged**